### PR TITLE
Upgrade sentry

### PR DESCRIPTION
**Pull request by theodur**
## Description
Upgrades Sentry to the latest version, `6.9.0`.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#27231

## Testing done
Tested on the `dev` environment with a [mock error](http://sentry.vfs.va.gov/organizations/vsp/issues/43024/?environment=development&environment=sandbox&query=&statsPeriod=14d).

## Screenshots
Screen Shot 2021-07-16 at 3 02 23 PM

## Acceptance criteria
- [ ] Sentry works properly with the latest version.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by timwright12 — APPROVED** [review]

File: src/platform/forms-system/test/js/review/SubmitController.unit.spec.jsx [hunk 1]
```diff
@@ -109,12 +109,10 @@ const createStore = (options = {}) => {
 };
 
 describe('Schemaform review: SubmitController', () => {
-  beforeEach(() => {
-    testkit.reset();
-  });
-  afterEach(() => {
+  before(() => {
```
Comment (theodur) [inline src/platform/forms-system/test/js/review/SubmitController.unit.spec.jsx:112]: I added a [comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/27231#issuecomment-881804204) about why this change is needed in the original ticket.

Comment (cvalarida) [reply src/platform/forms-system/test/js/review/SubmitController.unit.spec.jsx:112]: Like this, we're implicitly trusting that there's only one test in this `describe` block that uses `testkit`. :thinking: And if the problem is resetting it when it hasn't been used, I'm not sure that this won't run into issues later.

But then again, I don't expect we'll do much work on this before it's deprecated officially, so :man_shrugging: I guess.

**Review by cvalarida — APPROVED** [review]
I don't love the change to the `SubmitController` test, but it's fine for now to get this through. I may change it later.

File: src/platform/forms-system/test/js/review/SubmitController.unit.spec.jsx (hunk 1 above)
Comment (theodur) [reply src/platform/forms-system/test/js/review/SubmitController.unit.spec.jsx:112]: I completely agree, just not sure why it isn't working with the Sentry upgrade in this instance 🤔. If you do end up working on it and can get it working in the `beforeEach` and `afterEach` hooks, that'd be great!
